migrations/2024-11-21-195004_add_report_count/up.sql

```diff
@@ -77,3 +128,20 @@ FROM (
 WHERE
     a.comment_id = cnt.comment_id;
 
+-- Re-enable triggers after upserts
+ALTER TABLE comment_aggregates ENABLE TRIGGER ALL;
+
+-- Re-enable indexes
+UPDATE
+    pg_index
+SET
+    indisready = TRUE
+WHERE
+    indrelid = (
+        SELECT
+            oid
+        FROM
+            pg_class
+        WHERE
+            relname = 'comment_aggregates');
```

Comment:
Is this really safe? We should at least run reindex afterwards, or drop the indexes and recreate them afterwards.

https://stackoverflow.com/q/56628321